e2e_playwright/st_fragment_multiple_fragments.py

```diff
@@ -24,5 +26,13 @@ def my_fragment(n):
         st.write(f"uuid in fragment {n}: {uuid4()}")
 
 
+@st.fragment
+def exception_raising_fragment():
+    if st.checkbox("Raise Exception", value=raise_exception):
+        raise RuntimeError("This is an exception raised in a fragment")
+
+
 my_fragment(1)
+# fragment that raises an exception does not stop the flow
```

Comment:
nit: comment is now stale (or could be clarified a bit since it does stop execution depending on when the exception is raised)

Reply:
ahh great catch thanks 🙂 I have updated it!